ChangeLog.md

```diff
@@ -52,7 +52,8 @@
    ConstIterator and adds a new method to change the color of a
    specific face. (Bertrand Kerautret,
    [#937](https://github.com/DGtal-team/DGtal/pull/937))
-
+ - New methods to generate basic mesh of 3D tube and height field. New mesh module documentation.
```

Comment:
New methods to generate basic 3D tubular meshes and height fields. New mesh module documentation added. 

?